src/py/mat3ra/made/tools/analyze/lattice_planes.py

```diff
@@ -42,19 +42,28 @@ def get_pymatgen_slab_generator(
 
     @property
     def pymatgen_slab_generator_with_vacuum(self) -> PymatgenSlabGenerator:
-        return self.get_pymatgen_slab_generator()
+        return self.get_pymatgen_slab_generator(min_slab_size=self.DEFAULT_THICKNESS, min_vacuum_size=1)
 
     @property
     def pymatgen_slab_generator_without_vacuum(self) -> PymatgenSlabGenerator:
-        return self.get_pymatgen_slab_generator(min_vacuum_size=0)
+        return self.get_pymatgen_slab_generator(min_slab_size=1, min_vacuum_size=0)
 
     @property
     def all_planes_as_pymatgen_slabs_with_vacuum(self) -> List[PymatgenSlab]:
-        return self.pymatgen_slab_generator_with_vacuum.get_slabs(symmetrize=self.DEFAULT_SYMMETRIZE)
+        return [
+            *self.pymatgen_slab_generator_with_vacuum.get_slabs(symmetrize=not self.DEFAULT_SYMMETRIZE),
+            # Leaving option to create a non-symmetrized slabs too for additional terminations.
+            # As of now, looks like symmetrize=True gives all the unique terminations needed.
+            # *self.pymatgen_slab_generator_with_vacuum.get_slabs(symmetrize=self.DEFAULT_SYMMETRIZE),
```

Comment:
The whole point of having DEFAULTs is to use them as defaults - update DEFAULT_SYMMETRIZE to True and use here.